portfolio/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "test": "mocha --recursive ./src"
```

Comment:
per PR description, js tests are omitted. Should this file then also be omitted from this PR?